src/data/faq.json

```diff
@@ -2171,66 +2171,304 @@
+                },
+                {
+                    "term": "Release",
+                    "anchor": "release",
+                    "description": "Corona-Warn-App Version"
+                },
+                {
+                    "term": "Risk Encounter",
+                    "anchor": "risk_encounter",
+                    "description": "Encounter with a person who has potentially endangered oneself"
+                },
+                {
+                    "term": "Risk Calculation",
+                    "anchor": "risk_calculation",
+                    "description": "Four factors have an influence on the calculation of the individual risk of infection:<ul><li>Duration of the encounter</li><li>Distance at the time of the encounter</li><li>Time that has elapsed since the encounter</li><li>Infectiousness (derived from information on when and whether symptoms occur)</li></ul>If the calculated risk exceeds the threshold specified by the Robert Koch Institute, users will receive a notification regarding an increased risk."
+                },
+                {
+                    "term": "Risk Status",
+                    "anchor": "risk_status",
+                    "description": "A tile in the Corona-Warn-App whose color and content represent one's exposure to Corona infection. The status is composed of the encounters one has had with other people. If the risk of infection is low, the mentioned tile is green, if the risk is increased, the tile is red."
+                },
+                {
+                    "term": "Rolling Proximity Identifier (RPI)",
+                    "anchor": "rpi",
+                    "description": "IDs that ensure data protection and change regularly at short intervals, thus preventing the device from being tracked. These IDs can only be linked to likewise pseudonymous diagnostic keys ('positive identifiers') to perform encounter detection. Since these IDs change every 10 to 20 minutes, a person sitting next to you on the bus every day, for example, would not be able to associate you with an ID himself if the list of available RPIs were viewable."
                 }
             ],
             "S": [
+                {
+                    "term": "SARS-CoV-2",
+                    "anchor": "sars_cov_2",
+                    "description": "See <a href='#glossary_corona_virus'>Corona Virus</a>"
+                },
+                {
+                    "term": "Science-Blog",
+                    "anchor": "science_blog",
+                    "description": "The Corona-Warn-App Science Blog covers various scientific topics related to the Corona-Warn-App, such as its effectiveness, benefits, and users, as well as results of scientific evaluations conducted by the Robert Koch Institute."

```

Comment:
Suggest to add hyperlink to https://www.coronawarn.app/en/science/.

Reply:
Will be changed in the next commit, thanks!